src/main/java/nextstep/fp/Conditional.java

```diff
@@ -0,0 +1,5 @@
+package nextstep.fp;
+
+public interface Conditional {
+    boolean test(Integer number);
+}
```

Comment:
```suggestion
@FunctionalInterface
public interface Conditional {
    boolean test(Integer number);
}
```
함수형 인터페이스로 활용하고자 한다면 가능하면 `@FunctgionalInterface`를 명시해주는 것이 좋습니다. 생략가능하나 명시함으로써 해당 인터페이스에 메소드를 더 추가하지 말라라는 선언의 효과도 있어요 :)

Reply:
아하 말씀해주신 부분 참고해서 진행하도록 하겠습니다!